api/package.json

```
@@ -12,6 +11,10 @@
     "./build/esnext/platform/index.js": "./build/esnext/platform/browser/index.js",
     "./build/src/platform/index.js": "./build/src/platform/browser/index.js"
   },
+  "exports": {
+    ".": "./build/src/index.js",
+    "./experimental": "./build/src/experimental"
+  },
```

For completeness, I'd suggest listing all possible variants.

```suggestion
  "exports": {
    ".": {
      "module": "./build/esm/index.js",
      "esnext": "./build/esnext/index.js",
      "types": "./build/src/index.d.ts",
      "default": "./build/src/index.js"
    },
    "./experimental": {
      "module": "./build/esm/experimental.js",
      "esnext": "./build/esnext/experimental.js",
      "types": "./build/src/experimental.d.ts",
      "default": "./build/src/experimental.js"
    }
  },
```